File: server/Checkout Express/src/templates/CxSplitItem.java
package templates;

public class CxSplitItem {
	public static String run(Object id, boolean selected, int others, Object name, Object price, Object mods) {
		return	"<li class=\"'item "+(selected ? "selected":"")+" "+(others>0 ? "has-others":"")+"\"'>"+
			"\t<a itemID=\"'"+(id)+"\"'>"+
			"\t\t<span class=\"'name\"'>"+(name)+"</span>"+
			"\t\t<span class=\"'price\"'>"+(price)+"</span>"+
			"\t\t<ul class=\"'mods\"'>"+(mods)+"</ul>"+
			"\t</a>"+
			"\t<div class=\"'others\"'>"+
				(selected ? "Splitting with "+others+" other"+(others != 1 ? "s" : "") : others == 1 ? "Someone else is paying for this" : others+" others are splitting this" )+
			"</div>"+
			"</li>";
	}
}

File: misc/js/templates/cxSplitItem.js
var template = template || {};

template.cxSplitItem = function(id, selected, others, name, price, mods) {
	return	"<li class=\"item "+(selected ? "selected":"")+" "+(others>0 ? "has-others":"")+"\">\r\n"+
			"\t<a itemID=\""+(id)+"\">\r\n"+
			"\t\t<span class=\"name\">"+(name)+"</span>\r\n"+
			"\t\t<span class=\"price\">"+(price)+"</span>\r\n"+
			"\t\t<ul class=\"mods\">"+(mods)+"</ul>\r\n"+
			"\t</a>\r\n"+
			"\t<div class=\"others\">"+
				(selected ? "Splitting with "+others+" other"+(others != 1 ? "s" : "") : others == 1 ? "Someone else is paying for this" : others+" others are splitting this")+
			"</div>\r\n"+
			"</li>";
};